linode/manager
# refactor: [M3-7329] - MUI Migration - `SRC > Components > Crumbs`

## Description 📝

- MUI migration for the `Crumbs` component
- I really want to get this epic done so we can update to React 18 and drop @mui/styles

## Changes  🔄
- Runs codemod on `packages/manager/src/components/Breadcrumb/Crumbs.tsx`
- Find and replaces instances of `import { useTheme } from '@mui/styles';` to `import { useTheme } from '@mui/material';` because we intend to remove `@mui/styles` at the end of this migration
- Rewrites `Breadcrumb` Storybook story because the old one was broken

## Preview 📷

> [!note]
> No UI changes are expected

| before | after |
|--------|-------|
|     ![Screenshot 2023-10-25 at 3 14 26 PM](https://github.com/linode/manager/assets/115251059/b4830fb1-7fa0-4dfd-820c-b1e408c3b8aa)   |   ![Screenshot 2023-10-25 at 3 14 32 PM](https://github.com/linode/manager/assets/115251059/e37d656b-351e-497c-abeb-91eb61ac9e90)    |

## How to test 🧪

### Verification steps 
- Checkout this PR
- View various Breadcrumbs throughout Cloud Manager
- Verify Breadcrumbs look identical to what is at https://cloud.linode.com
- Optional: view the Storybook story

## As an Author I have considered 🤔

*Check all that apply*
